### ISSUE TITLE
Remove extra newlines in returned text

### ISSUE THREAD

**Issue by ehandal** [microsoft/pylance-release#1115]
I'm using coc-pyright and seeing text like this returned on textDocument/hover:
```
"contents": {
    "kind": "markdown",
    "value": "```python\n(variable) x: Literal[0]\n```\n"
},
```
and completionItem/resolve:
```
"documentation": {
    "kind": "markdown",
    "value": "```python\nclass ValueError()\n```\n"
}
```

This causes coc-pyright to show two line results on hover/completion like this:

![image](https://user-images.githubusercontent.com/23243879/113359563-a1013180-930d-11eb-9d9e-32344c33db60.png)

Not sure if coc-pyright should be expected to handle the newlines or strip them to properly display these on one line. If so, I could submit an issue there instead.

Using Pyright 1.1.126

**Comment by jakebailey** [comment]
This'll be fixed in the next version, and synced back to pyright by Wednesday.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.4.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202140-7-april-2021
